pixl_ehr/tests/conftest.py

```diff
@@ -34,11 +34,12 @@
 os.environ["EMAP_UDS_PASSWORD"] = "postgres"  # noqa: S105
 os.environ["EMAP_UDS_SCHEMA_NAME"] = "star"
 os.environ["COGSTACK_REDACT_URL"] = "test"
+os.environ["PROJECT_CONFIGS_DIR"] = (Path(__file__).parents[2] / "project_configs").as_posix()
```

Comment:
Why `.as_posix()` here? 🤔

Reply:
setting an env variable so it needs to be string. I have to admit the IDE suggested this method I didn't think too much of it - str(Path) makes more sense I guess?

Reply:
Yeah I'd say `str()` is more clear, and also consistent with what you did in the `pixl_core` conftests :) https://github.com/UCLH-Foundry/PIXL/blob/b55f7b3bd0683fd16313a179db2626b7727e17fc/pixl_core/tests/conftest.py#L45

Reply:
```suggestion
os.environ["PROJECT_CONFIGS_DIR"] = str(Path(__file__).parents[2] / "project_configs")
```